UnicornRender/video/source/CameraFpsController.cpp

```diff
@@ -67,11 +82,13 @@ void CameraFpsController::UpdateView(double posX, double posY)
 
     m_lastX = posX;
     m_lastY = posY;
-    if(m_dirty)
+
+    if(m_dirtyViewPosition)
```

Comment:
I can't find where `m_dirtyViewPosition` is set to `true` except constructor. Shall it be possible to reset this flag somehow?

Reply:
Will add ResetView function